ocs_ci/deployment/deployment.py

```diff
@@ -2037,6 +2040,29 @@ def get_and_apply_icsp_from_catalog(image, apply=True, insecure=False):
     return icsp_file_dest_location
 
 
+def get_latest_acm_tag_unreleased(version):
+    """
+    Get Latest tag for acm unreleased image
+
+     Args:
+        version (str): version of acm for getting latest tag
+
+    Returns:
+        str: image tag for the specified version, False when no tag found
+
+
+    """
+    response = requests.get(
+        "https://quay.io/api/v1/repository/acm-d/acm-custom-registry/tag/"
+    )
+    responce_data = response.json()
+    for data in responce_data["tags"]:
+        if version in data["name"] and "v" not in data["name"]:
+            return logger.info(data["name"])
```

Comment:
In [4]: type(logger.info("Hello"))
Out[4]: NoneType

This doesn't look good to me.

You should separate the log and return the data["name"] on separate line.

Reply:
I had added for my testing fixing in next comment